### PR TITLE
fix(*): fix seeking to frame boundary

### PR DESCRIPTION
**Pull request by stepancar**
I found issue, when we seek to boundary of frame (see test) it rounds integer with Math.floor, which seeks to previous frame in corner cases.

### CODE REVIEW

File: src/backends/beamcoder.ts [hunk 1]
```diff
@@ -239,7 +239,7 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
      */
     async getFrameAtTime(targetTime: number): Promise<beamcoder.Frame> {
         VERBOSE && console.log(`getFrameAtTime time(s)=${targetTime}`);
-        const targetPts = Math.floor(this._timeToPTS(targetTime));
+        const targetPts = Math.round(this._timeToPTS(targetTime));
```
Comment (stepancar) [inline src/backends/beamcoder.ts:242]: With Math.floor it seeks it 60, 60, 62 frames

File: test/framefusion.test.ts [hunk 2]
```diff
@@ -380,7 +410,7 @@ describe('FrameFusion', () => {
         // Act & assert
         // ensure we render the last few frames properly - we have to flush the decoder to get the last few frames
         for (let i = 50; i < 60; i++) {
-            const time = i / FPS + FRAME_SYNC_DELTA;
```
Comment (stepancar) [inline test/framefusion.test.ts:383]: FRAME_SYNC_DELTA guarantee us that we don't seek to boundary of frame.
Because with this PR I'm fixing seeking to boundary, I decided delete FRAME_SYNC_DELTA

**Review by animanathome — APPROVED** [review]

File: src/backends/beamcoder.ts (hunk 1 above)
Comment (animanathome) [reply src/backends/beamcoder.ts:242]: And now we get 60, 61 and 62

File: test/framefusion.test.ts (hunk 2 above)
Comment (antoineMoPa) [reply test/framefusion.test.ts:383]: Yeah that's a good change!

File: test/framefusion.test.ts [hunk 3]
```diff
@@ -137,7 +137,7 @@ describe('FrameFusion', () => {
 
         // Act and Assert
         for (let i = 0; i < 10; i++) {
-            const frame = await extractor.getFrameAtTime(i / FPS + FRAME_SYNC_DELTA);
+            const frame = await extractor.getFrameAtTime(i / FPS);
```
Comment (antoineMoPa) [inline test/framefusion.test.ts:140]: @stepancar question: is this equivalent to how we compute time in Luminary?

**Comment by antoineMoPa** [comment]
@stepancar can you bump version in package.json? it will help you later for package release.

File: src/backends/beamcoder.ts (hunk 1 above)
Comment (antoineMoPa) [reply src/backends/beamcoder.ts:242]: Nice find!

File: test/framefusion.test.ts [hunk 4]
```diff
@@ -354,7 +354,37 @@ describe('FrameFusion', () => {
         // Act & assert
         // ensure we render the last few frames properly - we have to flush the decoder to get the last few frames
         for (let i = 20; i < 30; i++) {
-            const time = i / FPS + FRAME_SYNC_DELTA;
+            const time = i / FPS;
```
Comment (antoineMoPa) [inline test/framefusion.test.ts:357]: I would have liked to see a few more tests with different FPS which are multiples of 5 up to 120, but maybe we should not worry about this until we have a case that actually proves that this is an issue 🤔

Comment (antoineMoPa) [reply test/framefusion.test.ts:357]: Maybe it's not even an issue, so just ignore my comment 😆

**Review by antoineMoPa — APPROVED** [review]
Nice!

File: test/framefusion.test.ts (hunk 3 above)
Comment (stepancar) [reply test/framefusion.test.ts:140]: @antoineMoPa , yes. The same time framefusion implementation shouldn't be aware of how it used in luminary.
I just fixed basic case when external code requests frame for time which points exactly to frame start in video.

Comment (antoineMoPa) [reply test/framefusion.test.ts:140]: Makes sense!
